<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>News</class>
 <widget class="QWidget" name="News">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="newsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>701</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NEWS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>99</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#34495e;color:white;margin:0px;text-align:center;padding:2px;border-radius:3px;width:100px;font-size:11px;font-weight:900;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/res/icons/update.png</normaloff>:/icons/res/icons/update.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="listSources">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>570</y>
     <width>256</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="newsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>701</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">a{color:#111 !important;text-transform:uppercase;font-weight:800;text-decoration:none !important;}</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
